app/components/collections/detail_component.rb

```diff
@@ -3,15 +3,7 @@
 
 module Collections
   # Renders the details section of the collection (show page)
-  class DetailComponent < ApplicationComponent
-    sig { params(collection: Collection).void }
-    def initialize(collection:)
-      @collection = collection
-    end
-
-    sig { returns(Collection) }
-    attr_reader :collection
-
+  class DetailComponent < Collections::ShowComponent
```

Comment:
Why this inheritance?  This doesn't seem to me like a natural inheritance relationship.